addons/gamepad/userscript.js

```diff
@@ -33,12 +33,10 @@ export default async function ({ addon, global, console, msg }) {
   const height = renderer._yTop - renderer._yBottom;
   const canvas = renderer.canvas;
 
-  const spacer = document.createElement("div");
```

Comment:
Separate PR please

Reply:
The change is for compatibility with the addon this PR adds.